templates/check-java.sh.erb

```diff
@@ -17,20 +17,18 @@ java_version=0
 
 if [[ $java_raw_version = *-ea* ]]
 then
-	# early access format e.g 11-ea
-    IFS='-' read -ra values <<< "$java_raw_version"
-    java_version=${values[0]}
+	# early access format e.g 11-ea or 11.0.18-ea
+    IFS='-' read -ra java_raw_version <<< "$java_raw_version"
```

Comment:
The java-11-openjdk-headless package in centos-stream8 ( used in acceptance tests ) report java version 11.0.18-ea.
These changes are needed for the script to resolv the major version correctly.
This issue and solution has been posted to atlassian support as well.

Reply:
https://jira.atlassian.com/browse/JRASERVER-77097